lk2nd/device/dts/msm8953/msm8953-xiaomi-ysl.dts

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8953 0>;
+	qcom,board-id = <QCOM_BOARD_ID_MTP 0>;
+};
+
+&lk2nd {
+	model = "Xiaomi Redmi S2/Y2 (ysl)";
+	compatible = "xiaomi,ysl";
+
+	panel {
```

Comment:
this looks like a leftover duplicate of msm8953-mtp.dts now

Reply:
Dropped from rules yet still on filesystem indeed.